src/rmc.rs

```diff
@@ -214,10 +208,14 @@ impl<'a, H: Signable + Hash + Eq + Clone + Debug, MK: MultiKeychain> ReliableMul
 
     /// Initiate a new instance of RMC for `hash`.
     pub async fn start_rmc(&mut self, hash: H) {
+        debug!(target: "AlephBFT-rmc", "starting rmc for {:?}", hash);
         let signed_hash = Signed::sign_with_index(hash, self.keychain).await;
+
         let message = Message::SignedHash(signed_hash.into_unchecked());
         self.handle_message(message.clone());
-        self.scheduler.add_task(Task::BroadcastMessage(message))
+        let task = Task::BroadcastMessage(message);
+        self.do_task(task.clone());
```

Comment:
Doing this task immediately is not as crucial as in the case of completing multisigned signature: this is veeery unlikely that we do not do the task immediately after running start_rmc, but we can keep it as is "just in case"

Reply:
Ok, let's keep it.